### PR TITLE
Do not show formerly attributed authors in item detail

### PR DESCRIPTION
**Pull request by eronisko**
WEBUMENIA-1593

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (link to Jira or GitHub issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/views/dielo.blade.php [hunk 1]
```diff
@@ -52,7 +52,17 @@
...
                     <h1 class="nadpis-dielo" itemprop="name">{{ $item->title }}</h1>
                     <h2 class="inline">
-                        @foreach($item->authorities->toBase()->merge($item->getAuthorsWithoutAuthority()) as $author)
+                        @php
+                            $authors = $item->authorities
+                                ->toBase()
+                                ->merge($item->getAuthorsWithoutAuthority())
+                                ->reject(function ($author) {
+                                    if (is_string($author)) return false;
+                                    return optional($author->pivot)->role === 'pôvodné určenie/formerly attributed to';
```
Comment (rastislav-chynoransky) [inline resources/views/dielo.blade.php:61]: also can `$author->pivot` ever be undefined?

File: resources/views/dielo.blade.php [hunk 2]
```diff
@@ -52,7 +52,17 @@
...
+                                });
```
[the remark this answers is not in the document]
Comment (eronisko) [reply resources/views/dielo.blade.php]: Great suggestion, changed.

File: resources/views/dielo.blade.php (hunk 1 above)
Comment (eronisko) [reply resources/views/dielo.blade.php]: Nope, good point. Changed.

**Review by igor-kamil — APPROVED** [review]
👌

**Pull request merged**